OWML.ModHelper/ModConfig.cs

```diff
@@ -30,8 +30,16 @@ public T GetSettingsValue<T>(string key)
 
             try
             {
-                var val = value is JObject obj ? obj["value"] : value;
-                return (T)Convert.ChangeType(val, typeof(T));
+                var objectValue = value as JObject;
+                var type = typeof(T);
+                var val = (objectValue != null) ? objectValue["value"] : value;
```

Comment:
seems this line is only used below the if?

Reply:
ah, no, not quite. still feel like this could be simplified. hard for me to say for sure via phone 😋

Reply:
in other words: it's a bit confusing to read. 
maybe better variable names could help. 
maybe inverting the if would help.
maybe moving the "val" variable below the if and fixing the resulting error in the if would help.

Reply:
and maybe extracting the enum thing to another method would help

Reply:
Would also be good to have less val / value variables, but I think that was already a problem in this settings code in general.